Backpack-SwiftUI/Price/Classes/BPKPrice.swift

```diff
@@ -142,7 +143,7 @@ public struct BPKPrice: View {
             items.append(leadingText)
         }
         
-        return alignment == .trailing ? items.reversed() : items
```

Comment:
Why is this changed? Was it on purpose?

Reply:
Ah yes, seems okay

Reply:
Yeah, in new design, the left and right alignmane will share the same order, so we remove the branch here
![image](https://github.com/Skyscanner/backpack-ios/assets/143154763/c33bbc14-2c9d-411e-99e8-0c8b171b0ddd)